dali/python/nvidia/dali/ops.py

```diff
@@ -234,11 +234,59 @@ def id(self):
 def _instantiate_constant_node(device, constant):
     return _Constant(device=device, value=constant.value, dtype=constant.dtype, shape=constant.shape)
 
+
+def _separate_kwargs(kwargs):
+    """Separates arguments into ones that should go to operator's __init__ and to __call__.
+
+    Returns a pair of dictionaries of kwargs - the first for __init__, the second for __call__.
+    """
+    def is_data_node(x):
+        return isinstance(x, _DataNode)
+    def is_call_arg(name, value):
+        if name == "device":
+            return False
+        if name == "name" or is_data_node(value):
+            return True
+        if isinstance(value, (str, list, tuple, nvidia.dali.types.ScalarConstant)):
+            return False
+        return not nvidia.dali.types._is_scalar_value(value)
+
+    def to_scalar(scalar):
+        return scalar.value if isinstance(scalar, nvidia.dali.types.ScalarConstant) else scalar
+
+    init_args = {}
+    call_args = {}
+    for name, value in kwargs.items():
+        if value is None:
```

Comment:
So we strip all arguments with None value?

Reply:
We already do it.